docs/tasks/semester-checklist.rst

```diff
@@ -3,7 +3,29 @@ Semester Checklist
 
 A quick checklist of tasks that need to be completed at the start of every semester.
 
--  Reserve the meeting room by speaking to the lab manager (requires a signature from the president and vice president). Get set up with swipe access if need be.
+Before Semester
+---------------
+Tasks which should be done during the break.
+
+-  Hold weekly/bi-weekly eboard meetings.
 -  Complete any outstanding eboard :doc:`transition tasks <../administration/eboard-onboarding-offboarding>`
--  Set up a meeting with the club faculty adviser to discuss plans for the semester
--  Update the website and thelink for the new semester
+-  Discuss plans and brainstorm for next semester.
+-  Plan for the Activities Gala (fall semester only).
+-  Reserve meeting space.
+
+At Start of Semester
+--------------------
+Tasks for last few weeks of break.
+
+-  Set up a meeting with the club faculty adviser to discuss plans for the semester.
+-  Create a poster for the start of the semester.
+-  Present at Activities Gala (fall semester only).
+-  Send out a welcome email to people from the gala and returning members.
+
+Post-Semester
```

Comment:
Post-Semester makes it seem like this is happening at the end of each semester. However, the comments underneath act like this is a few weeks into the semester. I suggest either adding these notes into the Start of Semester category, or rename this to reflect a couple weeks into the semester.